routes/api/resources.js

```diff
@@ -4,6 +4,7 @@ const _ = require('lodash');
 
 const router = express.Router();
 const models = require('../../models');
+const interceptors = require('../interceptors');
```

Comment:
@JesusEArambula I've added this import for the _interceptors_ helpers in interceptors.js. This is so that we can ensure login authentication on the create/update routes below...